modules/cloud-source-paladin-cloud.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-paladin-cloud.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="cloud-source-paladin-cloud_{context}"]
+= Configuring Paladin Cloud integration
+
+To discover cluster assets from Paladin Cloud, create a new integration in {product-title}.
+
+.Prerequisites
+* A Paladin Cloud account.
+* A Paladin Cloud API token.
+
+.Procedure
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Cloud source integrations* section and select *Paladin Cloud*.
+. Click *New Integration* (*`add`* icon).
+. Enter a name for *Integration name*.
+. Enter the Paladin Cloud API endpoint for *Paldin Cloud endpoint*. Defaults to https://api.paladincloud.io.
```

Comment:
One thing I'm a bit worried about at the moment: this API endpoint doesn't exist (alas, the domain name itself doesn't resolve yet with NXDOMAIN); worthwhile to check in with folks whether we should really advertise this as the API endpoint or be generic and say "your Paladin endpoint" (whatever this would be).

Reply:
I write another email to Paul to ask when this API will be available. Let's keep the default for now.